src/vivarium_public_health/metrics/disability.py

```diff
@@ -120,16 +121,27 @@ def disability_weight_aggregator(self, dw: pd.DataFrame) -> float:
 
     def report(
         self,
+        cause_state: Optional[DiseaseState],
```

Comment:
Doesn't the optional need to be the last arg or no?

Reply:
Nope, Optional inplicitly allows for None type. That is, `Optional[DiseaseState]` == `Union[DiseaseState, None]`